### PR TITLE
Add RSE TID/SID RNG to get shiny box eggs

### PR DESCRIPTION
**Pull request by zaksabeast**
# Guide Details

## I want to (select one)

- [x] Add a guide
- [ ] Modify an existing guide

## Games affected

- Ruby, Sapphire

## Type of RNG

TID/SID

## Language

English

## Notes

Guide donated by /u/OPBreloom

### CODE REVIEW

**Comment by zaksabeast** [comment]
This guide needs to be broken up into three parts

- Connect VBA to Dolphin
- TID/SID RNG for RSE
- Get shiny box eggs and use the first two guides

We may even want to leave the third part out and suggest it in the RSE TID/SID guide.

As such, this PR is here to preserve this branch, guide, and what needs to happen.

**Comment by zaksabeast** [comment]
That's another reason it should be separated into multiple guides - correction.

File: guides/rse/en/shiny box eggs (emu).md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Pokemon Box Shiny RNG (Emulator)
+_Original guide by /u/OPBreloom_
+
+### What you'll need for this RNG?
```
Comment (ShinySylveon04) [inline guides/rse/en/shiny box eggs (emu).md]: ? should be removed

File: guides/rse/en/shiny box eggs (emu).md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+- Latest Version of Dolphin
+- Latest Version of RNGReporter
+- Runasdate x32
+- One Ruby or Sapphire with one Save file after obtaining the Pokedex
+- One Ruby or Sapphire with a new game
+- RNG Tools (The one that has FindFrames etc.) Lua Scripts for Frames etc., of Pokemon Ruby/Sapphire
+- Pokemon Box
+
+### Dolphin's Configuration Tweaks
+First off, we'll need to open you Dolphin and adjust "Configuration":
```
Comment (ShinySylveon04) [inline guides/rse/en/shiny box eggs (emu).md]: typo - First off, we'll need to open Dolphin and adjust "Configuration":

File: guides/rse/en/shiny box eggs (emu).md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+- Make sure Dolphin has "Use Panic Handlers" disabled
+- Make sure Dolphin has "Pause on Focus Loss" disabled
+- In "Configure>Audio" only has "DSP HLE emulation (fast)" checked
+- In "Controllers" in the "Port 2" we'll want to input "GBA"m since we'll be using M-VBA for the connection.
```
Comment (ShinySylveon04) [inline guides/rse/en/shiny box eggs (emu).md]: typo?

File: guides/rse/en/shiny box eggs (emu).md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+### M-VBA's Tweaks
+For M-VBA, we'll need to go to "Emulation>Game Boy Advance" Click on "Real-Time Clock" and "Use BIOS File".
+
+Once you load a BIOS File we'll want to go to the "Emulation>GBA", and click on "Configure...".  Insert your "gba.bios" in "Boot ROM".  Uncheck "Skip BIOS" and "Pause when inactive".
+
+In "Option Settings", we'll want to click on "Link>Type", then "Gamecube", and in that same "Link" click on "Link at Boot" and "Speed Hack".
+
+### Pokemon Box/Dolphin Connecting with M-VBA
+Now turn on your Pokemon Box with Dolphin, and get to the point where once you've clicked "Pokemon Box" _NOT_ "Adventure".
```
Comment (ShinySylveon04) [inline guides/rse/en/shiny box eggs (emu).md]: needs to be rewritten to be more clear

File: guides/rse/en/shiny box eggs (emu).md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+Before turning on M-VBA, make sure you've gotten your Pokedex, and saved at a Pokemon Center. Turn on M-VBA, and Pokemon R/S.
+
+```
+Note: The game should be slow due to the connection through Dolphin - don't worry that's normal!
+```
+
+Once you're connected completely, M-VBA should have the Pokemon Box logo on the screen, and have entered Pokemon Box!
+
+You should be getting your first egg, which is False Swipe Swablu, we'll be making that shiny via TID/SID RNG on Pokemon R/S.
```
Comment (ShinySylveon04) [inline guides/rse/en/shiny box eggs (emu).md]: run-on sentence

File: guides/rse/en/shiny box eggs (emu).md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+Using a hex-to-dec calculator online, convert the first four characters of your Pokemon's PID from hex to decimal.  This will be your TID.
+
+Do the same with the last four characters of your Pokemon's PID from hex to decimal.  This will be your SID.
+
+Now we use the FindFrames. Input the PID and find your 16-bit Seed.  Open on RNGReporter and go to "Gen 3 Tools>16 bit seed to time".
+
+Input your 16-bit seed from Findframes.  With the date/time we have, we'll want to go to "Pandoras Box", and input the date/time as listed into "16-bit seed to time".
+
+We'll want to input our TID/SID and find the TID/SID Combo Frames lised.
```
Comment (ShinySylveon04) [inline guides/rse/en/shiny box eggs (emu).md]: typo - listed

File: guides/rse/en/shiny box eggs (emu).md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+```
+Note: If you don't find a frame within your TID/SID, check all 8 SID combonation, as that also can make the egg shiny.
```
Comment (ShinySylveon04) [inline guides/rse/en/shiny box eggs (emu).md]: typo - combinations

File: guides/rse/en/shiny box eggs (emu).md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+```
+
+### VBA-RR Pokemon R/S TID/SID RNG
+Before attempting this make sure VBA has:
+- "Real-Time Clock" checked
+- a BIOS File
+- has "Skip BIOS" disabled
+
+Run runasdate, and set the date like as given in the "16-bit seed to time".
```
Comment (ShinySylveon04) [inline guides/rse/en/shiny box eggs (emu).md]: reword this sentence

File: guides/rse/en/shiny box eggs (emu).md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+
+Click on "Immediate Mode" and "Move the time foward according to the real time".  Browse for the VBA-RR and click "Open".  After this, VBA should start running.  We'll want to open our second game without a save to RNG the TID/SID Combo.
+
+Play until the professor says "We'll see you in out Pokemon Lab", and wait for the target frame to appear.
```
Comment (ShinySylveon04) [inline guides/rse/en/shiny box eggs (emu).md]: typo

File: guides/rse/en/shiny box eggs (emu).md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+
+Don't worry - this is trial and error.Input the TID you got into "Pandoras Box", and exclude the SID/PID part.
+
+This will show you what frame you landed on.  Take your original target frame and subtract the frame you landed on from it to find your new target frame.
+
+Reload your game's savestate and repeat this process with your new target frame until the TID/SID combo is correct.
+
+### Hatching the Shiny Egg
+
+Play the game till you get you can trade between the two saves.
```
Comment (ShinySylveon04) [inline guides/rse/en/shiny box eggs (emu).md]: reword

**Comment by zaksabeast** [comment]
We're not merging this, it's only here to preserve the fact that this needs to be broken up into two guides.

**Comment by zaksabeast** [comment]
This guide became #16 and was inspiration to ask /u/Upper90175 for #19.  As such, this pull request will now be closed.

**Pull request closed without merging**